receiver/cudareceiver/README.md

```diff
@@ -0,0 +1,26 @@
+# CUDA (GPU) Receiver
+
+**Status: alpha**
+
+## Prerequisites
+
+CUDA Receiver is currently only supporte on the following environment.
+
+* OS: Ubuntu 18.04
+* CPU architecture: x86_64
+* CUDA Toolkit: version 10.2
+
+Technically, it depends upon `nvml.h` and `libnvidia-ml.so`, and it should work on the environment wher CUDA Toolkit version 10.2 is available.
```

Comment:
```suggestion
Technically, it depends upon `nvml.h` and `libnvidia-ml.so`, and it should work on the environment where CUDA Toolkit version 10.2 is available.
```

Can you clarify, `nvml.h` is required at compile time and `libnvidia-ml.so` is required at runtime I guess?

Reply:
No, libnvidia-ml.so is required on build time as well.